accept-leads-xml.md

```
@@ -1,68 +1,109 @@
-## XML Posting Guidelines
+# XML Posting Guidelines
 
 This document provides instructions on implementing an XML feed that may be used to immediately notify your CRM system of a lead. The following information outlines the format of the XML that will be sent to you.
 
+You may receive more elements than listed below in a POST;
+
 ### Possible Attributes and Values
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)